python/py-avro/Portfile

```diff
@@ -17,18 +16,27 @@ description         Avro is a serialization and RPC framework.
 long_description    ${description}
 
 homepage            https://avro.apache.org/
+master_sites        pypi:a/avro
+distname            avro-${version}
 
-checksums           rmd160  61c45d1cfa9914306401cb88d880ac63602d43cc \
-                    sha256  f456539183df0d6912daedfbf0f94d7cf69b4fba7200490e8a06d1e8f23af43d \
-                    size    1845163
+checksums           rmd160  f7465dfc3a5bbdd27265617d3039591313eefc82 \
+                    sha256  bbf9f89fd20b4cf3156f10ec9fbce83579ece3e0403546c305957f9dac0d2f03 \
+                    size    67793
 
 python.versions     37 38
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                        port:py${python.version}-setuptools
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-codestyle \
+                    port:py${python.version}-isort
 
-    worksrcdir          ${distname}/lang/py3
+    test.run        yes
```

Comment:
you are missing a test-dependency on `py-pytest`

Reply:
Thanks done.